feat(AWS Lambda): Support `ephemeralStorageSize` (#10901)
serverless/serverless: docs/providers/aws/guide/functions.md

        - subnetId1
```

> ## Size of available ephemeral storage

Reviewer: ```suggestion
## Ephemeral storage
```

(for simplicity's sake?)

Author: 👍

> You can configure size of the ephemeral storage available to your Lambda function under `/tmp` directory by setting the `ephemeralStorageSize` property in the function configuration in `serverless.yml`. It should be a numerical value of MBs, between 512 and 10240. If you don't specify any configuration, the function will have 512 MB ephemeral storage available by default.

Reviewer: ```suggestion
By default, Lambda [allocates 512 MB of ephemeral storage](https://docs.aws.amazon.com/lambda/latest/dg/configuration-function-common.html#configuration-ephemeral-storage) in functions under the `/tmp` directory.

You can increase its size via the `ephemeralStorageSize` property. It should be a numerical value in MBs, between 512 and 10240.
```

Reviewer: (I reused some of the wording in https://docs.aws.amazon.com/lambda/latest/dg/configuration-function-common.html#configuration-ephemeral-storage)

Author: 👍
